sympy/combinatorics/tests/test_homomorphisms.py

```diff
@@ -83,6 +84,9 @@ def test_isomorphisms():
     assert check
     assert T(b*a*b**-1*a**-1*b**-1) == Permutation(0, 2, 3)
     assert T(b*a*b*a**-1*b**-1) == Permutation(0, 3, 2)
+    # Find all possible epimorphisms.
+    list_hom = find_homomorphism(G, H, surjective=True, all=True)
+    assert all(elem.is_surjective() for elem in list_hom)
```

Comment:
Would be good to have a couple more `find_homomorphism` tests that don't just look for isomorphisms. Maybe something injective, and you should also try `compute=False`. The more features you cover the better